EscapeRoom/src/main/java/utils/MenuUserOptions.java

```diff
@@ -5,7 +5,7 @@ public class MenuUserOptions {
     static final String m2 = "Subscribe / Unsubscribe User";
     static final String m3 = "Display user certificates";
     static final String m4 = "Display user gifts";
-    static public final String close = "Close.";
+    static public final String close = "Back to main menu";
```

Comment:
Reminder para alguno de los 2: Hay que cambiar también el Close por "Back to main menu" en el Menu de delete, para que todo quede congruente.